fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/serialization/LoanApplicationTransitionValidator.java

```diff
@@ -237,6 +283,14 @@ public void validateAccountStatus(final Loan loan, final LoanEvent event) {
                     dataValidationErrors.add(error);
                 }
             }
+            case LOAN_WITHDRAWN -> {
```

Comment:
Is this used anywhere?

Reply:
Will remove this from method. It is not used anywhere. This check is taken care with status transition check (line 242).